_extensions/godot_descriptions.py

```diff
@@ -103,10 +103,11 @@ def generate_description(app, pagename, templatename, context, doctree):
     doctree.walkabout(generator)
 
     description = (
-        '<meta name="description" content="' + generator.create_description() + '">\n'
+        '<meta name="description" content="' + generator.create_description() + '" />\n'
```

Comment:
Void elements like `meta` should only specify a start tage, but not self-close: https://html.spec.whatwg.org/#void-elements

The common recommendation everywhere seems to be to just use `<meta>`, and outside of XHTML that always was valid. The majority of the web content out there seems to do this. Does this specific thing really break EPUB? That seems weird.

See also this discussion: https://stackoverflow.com/questions/4693939/self-closing-tags-void-elements-in-html5

Reply:
Per the [W3C Recommendations on HTML5](https://www.w3.org/TR/2014/REC-html5-20141028/syntax.html) section on [Start tags](https://www.w3.org/TR/2014/REC-html5-20141028/syntax.html#start-tags), void elements can be self-closing (which appears to be different than having an end-tag). Also, the [examples for `meta` on developer.mozilla.org](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#examples) are self-closing.

> The common recommendation everywhere seems to be to just use <meta>, and outside of XHTML that always was valid. The majority of the web content out there seems to do this. Does this specific thing really break EPUB? That seems weird.

Per the w3c epub docs, the [epub format is XHTML](https://idpf.org/epub/30/spec/epub30-contentdocs.html#sec-overview-relations-html) that is ["a profile of HTML5"](https://idpf.org/epub/30/spec/epub30-contentdocs.html#sec-xhtml).

Reply:
Thanks for the detailed reponse, that makes sense. :)